data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,41 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
+    "unmanagedWrappingKeys": {
+      "description" : "Key derivation parameters for unmanaged wrapping keys",
+      "type": "object",
+      "default": {},
+      "patternProperties": {
+        ".*": { 
+          "type" :  "object",
+          "properties": {
+            "salt": {
+              "description": "Salt for the key derivation function",
+              "type": "string"
+            },
+            "passphrase": {
+              "description": "Passphrase for the key derivation function",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "defaultUnmanagedWrappingKey": {
```

Comment:
I think maybe it would be clearer if this was called `defaultRootWrappingKeyId`.

And what' the thinking about the rotation when it's added? that we add a `previousRootWrappingKeyId`?

Reply:
Let's rename to `defaultRootWrappingKey`